esprima.js

```diff
@@ -3616,6 +3692,10 @@
         startToken = lookahead;
         token = lookahead;
 
+        if (matchKeyword('yield')) {
```

Comment:
> The spec outlines the rules for it, but yield is a contextual keyword/identifier.

OK, let's continue at #1186 so that it is not buried/forgotten easily.